platform/jvm/capture/src/main/kotlin/io/bitdrift/capture/attributes/NetworkAttributes.kt

```diff
@@ -88,7 +90,12 @@ internal class NetworkAttributes(
                 connectivityManager.activeNetwork?.let { network ->
                     updateNetworkType(connectivityManager.getNetworkCapabilities(network))
                 }
-                connectivityManager.registerDefaultNetworkCallback(this)
+                if (Build.VERSION.SDK_INT < Build.VERSION_CODES.N) {
```

Comment:
does this mean <23 devices will not receive networking info at all? sounds like we should fallback to the more expensive polling logic in these cases